lib/rebuild.js

```diff
@@ -27,7 +27,7 @@ class Rebuild extends BaseCommand {
 
   async rebuild (args) {
     const globalTop = resolve(this.npm.globalDir, '..')
-    const where = this.npm.flatOptions.global ? globalTop : this.npm.prefix
+    const where = this.npm.config.get('global') ? globalTop : this.npm.prefix
```

Comment:
~Suggest naming this `path` for consistency, since it's only used once on the subsequent line, and it is the `path` option.~ I guess that can happen later, since all the other install friends do this.  Nevermind.